crates/iceberg/Cargo.toml

```diff
@@ -62,4 +62,5 @@ uuid = { workspace = true }
 [dev-dependencies]
 pretty_assertions = { workspace = true }
 tempfile = { workspace = true }
+tera = { workspace = true }
```

Comment:
What is this used for?

Reply:
`tera` is a template engine in rust. We need to mock up some test data to do unittest. For example, in `crates/iceberg/testdata/example_table_metadata_v2.json`, we need to setup the real table location, manifest file path, etc using temporary dir, which is only available in runtime.

Reply:
Or by just integrating with the existing REST catalog, that will take care of all the paths and such :)

Reply:
I agree that we will eventually add these into integration tests, but I also love small unit tests: they are fast to run, easy to debug, easy to understand.

Reply:
I'll add integration tests later to verify it. But I think unit test is still quite valuable for ensuring quality. No worry, the `dev.dependency` is sth similar to `test` scope java's build tools like maven or gradle, e.g. it will only be used in tests, and will be excluded when linked to production code.

Reply:
Thanks for the context. What I would like about the integration tests, is that is also very easy for me to make test-cases :)